sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,7 @@
+select count(*),
+        "c"."name"
+  from "castMembers"
+join "filmCategory" using ("filmId")
+join "categories" as "c" using ("categoryId")
+where "actorId" = '178'
```

Comment:
In this case, the constraint is the actor's name. Even if you know their id, you can't guarantee when it runs later that they'll have that same id, so it'd be better to filter by their name.

---

![image](https://user-images.githubusercontent.com/16143771/144960202-05a75cb7-b313-4600-bc6c-2b364832b2be.png)